### PR TITLE
Project documentation - Update

### PR DESCRIPTION
**Pull request by GabrielaNitu**
### Pull Request Overview

This pull request adds the updates for documentation.

### Testing Strategy

This pull request was tested by Nitu-Sararu Ramona-Gabriela

### TODO or Help Wanted

This pull request still needs the approval.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Nitu-Sararu Ramona-Gabriela ramona.nitu@stud.fils.upb.ro

### CODE REVIEW

File: docs/project/2024/ramona.nitu/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## Motivation
+
+I chose this project because of its relevance in various emergency aspects, such as search and rescue operations and communication in remote areas. Morse code remains a fundamental skill in these contexts, offering a reliable means of communication when other methods may be unavailable. Additionally, undertaking this project provides a valuable opportunity for personal learning and skill development. Mastering Morse code not only equips me with practical skills for emergency situations but also deepens my understanding of communication systems and encoding methods.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
+
```
Comment (TeodoraMiu) [inline docs/project/2024/ramona.nitu/index.md:23]: You can embed the architecture image here, something like this:
`architecture`

**Review by TeodoraMiu — DISMISSED** [review]

File: docs/project/2024/ramona.nitu/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+## Motivation
+
+I chose this project because of its relevance in various emergency aspects, such as search and rescue operations and communication in remote areas. Morse code remains a fundamental skill in these contexts, offering a reliable means of communication when other methods may be unavailable. Additionally, undertaking this project provides a valuable opportunity for personal learning and skill development. Mastering Morse code not only equips me with practical skills for emergency situations but also deepens my understanding of communication systems and encoding methods.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
```
Comment (alexandruradovici) [inline docs/project/2024/ramona.nitu/index.md:22]:
```suggestion
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/ramona.nitu/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Multi-Sensory Morse Code Translator
+
```
Comment (alexandruradovici) [inline docs/project/2024/ramona.nitu/index.md:2]: Please add a one line description of your project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/ramona.nitu/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+:::info 
+
+**Author**: Nițu-Săraru Ramona-Gabriela
```
Comment (alexandruradovici) [inline docs/project/2024/ramona.nitu/index.md:5]:
```suggestion
**Author**: Nițu-Săraru Ramona-Gabriela \
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/ramona.nitu/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+## Motivation
+
+I chose this project because of its relevance in various emergency aspects, such as search and rescue operations and communication in remote areas. Morse code remains a fundamental skill in these contexts, offering a reliable means of communication when other methods may be unavailable. Additionally, undertaking this project provides a valuable opportunity for personal learning and skill development. Mastering Morse code not only equips me with practical skills for emergency situations but also deepens my understanding of communication systems and encoding methods.
+
+## Architecture 
+
+1) Text Input Provided to the System: The user enters text input through a connected input device, such as a keyboard. The input text is then passed to the Morse code translator for conversion.
```
Comment (alexandruradovici) [inline docs/project/2024/ramona.nitu/index.md:21]: Does this render correctly? Did you run `npm start` and looked at it?

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/ramona.nitu/index.md (hunk 5 above)
Comment (GabrielaNitu) [reply docs/project/2024/ramona.nitu/index.md:21]: Yes, I used npm start and it looked ok.

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**